.pre-commit-config.yaml

```diff
@@ -1266,6 +1267,14 @@ repos:
         files:
           ^airflow-ctl/src/airflowctl/api/operations.py|airflow-ctl/src/airflowctl/cli/.*\.py$
         additional_dependencies: ['rich>=12.4.4']
+      - id: mockery
+        name: Generate mocks for go
+        entry: --entrypoint ./scripts/in_container/run_mockery.sh vektra/mockery:3
```

Comment:
I think in this case it would be clearer to just have `mockery` command etc here. The "in_container" is for when the commands do more things, but here what we are running is very simple, so having it direct is much much easier to understandd.